comments/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils import timezone
+from django.conf import settings
+
+class CommentManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted_at=None)
+    
+class Comment(models.Model):
+    member = models.ForeignKey("members.Member", on_delete=models.CASCADE)
+    content = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+    deleted_at = models.DateTimeField(null=True)
+    like_comment = models.ManyToManyField(settings.AUTH_USER_MODEL, through="LikeComment", related_name="like_comment")
+    
+    objects = CommentManager()
+    
+    def delete(self):
+        self.deleted_at = timezone.now()
+        self.save()
+        
+class LikeComment(models.Model):
+    like_comment = models.ForeignKey("Comment", related_name="comment", on_delete=models.CASCADE)
+    like_sender = models.ForeignKey(settings.AUTH_USER_MODEL, related_name="sender", on_delete=models.CASCADE)
```

Comment:
```suggestion
    like_sender = models.ForeignKey(settings.AUTH_USER_MODEL, related_name="sender", on_delete=models.CASCADE)
```
like_sender 有點怪，like_by 可能好一點

Reply:
好的 再做調整